src/components/createCluster/validationSchema.ts

```diff
@@ -0,0 +1,20 @@
+import * as Yup from 'yup';
+
+export default Yup.object().shape({
+  clusterName: Yup.string().required('Required'),
+  DNSDomain: Yup.string().required('Required'),
+  pullSecret: Yup.string().when('$providePullSecret', {
+    is: true,
+    then: Yup.string().required('Required')
+  }),
+  username: Yup.string().when('$providePullSecret', {
+    is: false,
+    then: Yup.string()
+      .required('Required')
+      .email('Incorrect format, please provide email')
```

Comment:
My username isn't an email. I wasn't able to submit the form without tweaking this.

Reply:
Removed email validation for now